File: src/main/java/loop/LoopAPlusB.java
package loop;

import java.util.Scanner;

public class LoopAPlusB {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int number = scanner.nextInt();

        for (int i = 0; i < number; i++) {
            int a = scanner.nextInt();
            int b = scanner.nextInt();
            System.out.println(a + b);
        }

        scanner.close();
    }

}

File: src/main/kotlin/loop/LoopAPlusB.kt
package loop

import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    val number: Int = scanner.nextInt()

    for (i in 1..number) {
        val a: Int = scanner.nextInt()
        val b: Int = scanner.nextInt()
        println(a + b)
    }

    scanner.close()
}